lib/library.rb

```diff
@@ -0,0 +1,68 @@
+class Library
+
+  attr_reader :name, :books, :authors, :checked_out_books
+
+  def initialize(name)
+    @name = name
+    @books = []
+    @authors = []
+    @checked_out_books = []
+    @book_checkout_count = {}
+  end
+
+  def add_author(author)
+    @authors << author
+    author.books.each do |book|
+      @books << book
+    end
```

Comment:
good job here, even simpler for this `.each` call:
```@books + author.books`
Ruby allows arrays to be merged with the `+` operator

Reply:
![wat](https://user-images.githubusercontent.com/51726839/64308661-42b87500-cf57-11e9-9b88-3437b7bdaa01.gif)